package.json

```diff
@@ -112,10 +113,12 @@
     "source-map-loader": "5.0.0",
     "style-loader": "4.0.0",
     "terser-webpack-plugin": "5.3.10",
+    "tinybench": "^3.0.3",
     "ts-jest": "29.2.5",
     "ts-loader": "9.5.1",
     "ts-node": "^10.9.1",
     "tsconfig-paths": "^4.2.0",
+    "tsx": "^4.19.2",
```

Comment:
Nice! :star_struck: We already have `ts-node` as a typescript runner, but [I looked into it](https://github.com/privatenumber/ts-runtime-comparison) and `tsx` looks a lot better/more promising. I'll look into replacing our few usages of `ts-node` with `tsx` after this.

Reply:
oo didn't even notice, used `tsx` because it was part of the example code of tinybench